pkgs/by-name/tr/trashy/package.nix

```diff
@@ -19,12 +19,16 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+  postInstall = ''
+    $out/bin/trash manpage > trashy.1
```

Comment:
This needs to be hidden behind the `lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform` guard as well, since it's running the executable.